swarm/CHANGELOG.md

```diff
@@ -5,9 +5,70 @@
 - Export `NetworkBehaviour` derive as `libp2p_swarm::NetworkBehaviour`.
   This follows the convention of other popular libraries. `serde` for example exports the `Serialize` trait and macro as
   `serde::Serialize`. See [PR 3055].
+
 - Feature-gate `NetworkBehaviour` macro behind `macros` feature flag. See [PR 3055].
 
+- Make executor in Swarm constructor explicit. See [PR 3097].
```

Comment:
Thank you for the great changelog entry.

Reply:
Thanks :)